src/adapters/bidderFactory.js

```diff
@@ -66,6 +66,7 @@ import { logWarn, logError, parseQueryStringParameters, delayExecution } from 's
  * @property {('GET'|'POST')} method The type of request which this is.
  * @property {string} url The endpoint for the request. For example, "//bids.example.com".
  * @property {string|object} data Data to be sent in the request.
+ * @property {object} options Content-Type set in the header of the bid request, overrides default 'text/plain'.
```

Comment:
Proposed changes to core modules are welcome but should be made in a separate PR. Also calls from 1.0 bidders will have `contentType: 'text/plain'` set by default, so needing to set `options: {contentType: 'text/plain'}` on the object returned by your `buildRequests` function is unnecessary I believe?

Reply:
yeah sorry I couldn't figure out why that was happening either.